### PR TITLE
feat(zkstack_cli): Add status page

### PR DESCRIPTION
**Pull request by matias-gonz**
## What ❔

Sample:

image

### CODE REVIEW

**Review by Deniallugo — DISMISSED** [review]

File: zkstack_cli/crates/zkstack/src/commands/dev/commands/status.rs [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+            }
+        })
+        .collect::<Vec<String>>()
+        .join(" ")
+}
+
+pub async fn run(shell: &Shell, args: StatusArgs) -> anyhow::Result<()> {
+    if args.ports {
+        return print_ports(shell);
+    }
```
Comment (Deniallugo) [inline zkstack_cli/crates/zkstack/src/commands/dev/commands/status.rs:243]: let's have it as a subcommand, not an arg

File: zkstack_cli/crates/zkstack/src/commands/dev/commands/status.rs
Comment (Deniallugo) [inline zkstack_cli/crates/zkstack/src/commands/dev/commands/status.rs:1]: nit. Can we split this into: 
1. Command running
2. Drawing 
3. Utils (such as port openings)

File: zkstack_cli/crates/zkstack/src/commands/dev/commands/status.rs (hunk 1 above)
Comment (matias-gonz) [reply zkstack_cli/crates/zkstack/src/commands/dev/commands/status.rs]: https://github.com/matter-labs/zksync-era/pull/3036/commits/0d89c8ea8faaa65c25b67ecb2b0ebe2598b1cbdb

File: zkstack_cli/crates/zkstack/src/commands/dev/commands/status.rs
Comment (matias-gonz) [reply zkstack_cli/crates/zkstack/src/commands/dev/commands/status.rs:1]: Good idea, i will split the code

**Review by Deniallugo — APPROVED** [review]

**Pull request merged**